modules/overview/pages/introduction-to-eclipse-che.adoc

```diff
@@ -11,7 +11,7 @@
 * A multi-container workspace for each developer with the ability to replicate with a single click using {prod} factories.
 * Pre-built stacks with the ability to create custom stacks for any language or runtime.
 * An enterprise integration using OpenShift OAuth or Dex.
-* Browser-based IDEs; integration with Che-Theia or any other web IDE, such as Jupyter.
+* Browser-based IDEs; integration with link:https://github.com/che-incubator/che-code[Microsoft Visual Studio Code] and others such as link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains IntelliJ IDEA Community Edition].
```

Comment:
We already have terminology for these IDE names in the docs:
https://www.eclipse.org/che/docs/stable/end-user-guide/selecting-a-workspace-ide/
So I suggest sticking with one common terminology for now.
Or, if you're not happy with reentering these manually, you can add new attributes for these in the upstream and downstream repos and then update the docs with the attributes.
If in doubt, you can double-check these IDE names with Nick on the forum channel.

```suggestion
* Browser-based IDEs; integration with link:https://github.com/che-incubator/che-code[Microsoft Visual Studio Code - Open Source] and others such as link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains IntelliJ IDEA Community Edition].
```